File: challenges/2-idade-em-dias/javascript/heltonp21/idadeDias.js
let idade = process.argv[2]

const anos = parseInt(idade / 365)
const meses = parseInt((idade % 365) / 30)
const dias = (idade % 365) % 30

console.log(`
${anos} ano(s)
${meses} mes(es)
${dias} dia(s)
`)

File: challenges/2-idade-em-dias/java/herco/IdadeDias.java
import java.util.Scanner;

public class IdadeDias {
    final static int ANO = 365;
    final static int MES = 30;

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Entre com sua idade em dias: ");
        int idadeEmDias = scanner.nextInt();

        int anos = idadeEmDias / ANO;
        int meses = (idadeEmDias % ANO) / MES;
        int dias = (idadeEmDias % ANO) % MES;

        System.out.println(anos + " ano(s)");
        System.out.println(meses + " mes(es)");
        System.out.println(dias + " dia(as)");

        scanner.close();
    }
}